DEV/i18nextStateAsync.js

```diff
@@ -0,0 +1,36 @@
+import i18n from 'i18next'
+import HttpApi from 'i18next-http-backend'
+import { factoryI18nextState } from '../src'
+
+i18n
+  .use(HttpApi)
+  .init({
+    fallbackLng: 'en',
+    fallbackNS: 'translation',
+    languages: ['en', 'pt-BR', 'es'],
+    lng: 'en',
+    // namespaces: ['translation', 'mobile'],
+    /**
+     * Important to load only the language required
+     * e.g. loading 'pt-BR', it won't load 'pt'
+     */
+    load: 'currentOnly',
```

Comment:
I believe it should be set by default in `i18next`. requesting `pt-BR` and `i18next` fetch data for `pt` and `pt-BR` does not make sense, as it is triggered by a literal request of `changeLanguage`.